CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing to the LaunchDarkly JavaScript Observability SDK
+
+LaunchDarkly has published an [SDK contributor's guide](https://docs.launchdarkly.com/sdk/concepts/contributors-guide) that provides a detailed explanation of how our SDKs work. See below for additional information on how to contribute to this SDK.
+
+## Submitting bug reports and feature requests
+
+The LaunchDarkly SDK team monitors the [issue tracker](https://github.com/launchdarkly/observability-sdk/issues) in the SDK repository. Bug reports and feature requests specific to this library should be filed in this issue tracker. The SDK team will respond to all newly filed issues within two business days.
+
+## Submitting pull requests
+
+We encourage pull requests and other contributions from the community. Before submitting pull requests, ensure that all temporary or unintended code is removed. Don't worry about adding reviewers to the pull request; the LaunchDarkly SDK team will add themselves. The SDK team will acknowledge all pull requests within two business days.
+
+## Build instructions
+
+### Prerequisites
+
+`npm` or `yarn` is required to develop in this repository.
+
+We recommend installing the [LaunchDarkly JavaScript SDK](https://github.com/launchdarkly/js-client-sdk) before developing in this repository.
+
+### Setup
+
+To install project dependencies:
+
+```
+# npm
+npm i @launchdarkly/observability
+npm i @launchdarkly/session-replay
+
+# yarn
+yarn add @launchdarkly/observability
+yarn add @launchdarkly/session-replay
+
+```
+
+### Build
+
+To build all projects, from the root directory:
+
+```
+yarn build
+```
+
+Running `yarn build` in an individual package (`@launchdarkly/observability` or `@launchdarkly/session-replay`) will build that package, but will not rebuild any dependencies.
+
+### Testing
+
+To run all unit tests:
+
+```shell
+TKTK
+```
+
+It is preferable to run tests against TKTK-which versions prior to submitting a pull request. However, LaunchDarkly's CI tests will run automatically against all supported versions.
```

Comment:
which versions?

or, we can remove this line if there's not appropriate CI yet.

Reply:
we don't test our SDK against different versions at the moment. Could point to the `.github/workflows/turbo.yml` file for a reference to what CI runs